packages/palette/src/elements/Input/Input.tsx

```diff
@@ -142,6 +154,25 @@ const StyledInput = styled.input<StyledInputProps>`
         cursor: default;
         ${INPUT_STATES.disabled}
       }
+
+      ${props.title &&
+      css`
+        ::placeholder {
+          opacity: 0;
+        }
+      `}
     `
   }};
 `
+
+const StyledLabel = styled.label<StyledInputProps>`
+  position: absolute;
+  top: 50%;
+  left: 5px;
+  padding: 0 5px;
+  background-color: ${themeGet("colors.white100")};
+  transform: translateY(-50%);
+  transition: 0.2s cubic-bezier(0.65, 0.05, 0.36, 1);
```

Comment:
Same question regarding spec. Where is this curve coming from?

Reply:
We don't want to `transition` *all* properties; so make sure you add `transform` here.

Reply:
For the curve, I was playing around with [this tool](https://cubic-bezier.com) to be as close to [the material design guidelines](https://m3.material.io/styles/motion/easing-and-duration/applying-easing-and-duration#d4348fa7-87b9-4dda-be70-c126111fd63c) 

But for the transition, I'll specify the properties that are going to be changed :) (font size, color, left, top, etc)

Reply:
Wondering if we can do this label position transition using *only* `transform`, actually? There's no real reason we couldn't. See: https://web.dev/animations-guide/#layout

Reply:
I managed to update the size (using `scale`) and the position (using `translate(x , y)`), but I'm not sure how to animate the changing of the text color with transform, do you know how we could do that?

In the meantime, I'll update the transition property to only update the animating ones, not _all_

Reply:
Nice — you *could* get clever using `opacity` since that is also one of the properties that's hardware accelerated but for now I'd continue to just transition on `color`. We might want to tag these with [`will-change`](https://developer.mozilla.org/en-US/docs/Web/CSS/will-change), but it'd probably be good to just measure and optimize later.

I think these animations are fast enough and nothing else generally will be happening on screen as far as layout goes so I'm not anticipating this being much of a problem. Just, in general, if something can be achieved with a `transform` then we should do that. 👍